cli/main.go

```diff
@@ -10,10 +12,19 @@ import (
 
 const sentryFlushTimeout = 5 * time.Second
 
+//nolint:gocritic
```

Comment:
Wouldn't add nolint to the whole fn but to the os.Exit line?

Reply:
done